server/src/com/thoughtworks/go/server/web/IgnoreResolver.java

```diff
@@ -76,6 +77,10 @@ private boolean isGoingToAbout(HttpServletRequest request) {
         return StringUtils.contains(request.getRequestURI(), request.getContextPath() + "/about");
     }
 
+    private boolean isGoingToPlugin(HttpServletRequest request) {
+        return StringUtils.contains(request.getRequestURI(), request.getContextPath() + "/plugins/interact");
```

Comment:
ya i forgot to reply to @ketan's question. its not listed on this page for some reason.

either way is fine by me. spring seemed easier to get through to for spike.